samples/CommunityToolkit.Maui.Sample/MauiProgram.cs

```diff
@@ -20,75 +25,156 @@ public static MauiApp CreateMauiApp()
 		builder.Services.AddHttpClient<ByteArrayToImageSourceConverterViewModel>();
 
 		// Add Gallery View Models
-		builder.Services.AddTransient<AlertsGalleryViewModel>();
-		builder.Services.AddTransient<BehaviorsGalleryViewModel>();
-		builder.Services.AddTransient<ConvertersGalleryViewModel>();
-		builder.Services.AddTransient<ExtensionsGalleryViewModel>();
-		builder.Services.AddTransient<LayoutsGalleryViewModel>();
-		builder.Services.AddTransient<ViewsGalleryViewModel>();
+		RegisterPages(builder.Services);
+		RegisterViewModels(builder.Services);
+
+		return builder.Build();
+	}
+
+	static void RegisterPages(IServiceCollection services)
```

Comment:
Let's add `in` to ensure a future PR doesn't change `IServiceCollection`

```suggestion
	static void RegisterPages(in IServiceCollection services)
```